controllers/horizontest_controller.go

```diff
@@ -60,7 +60,7 @@ type HorizonTestReconciler struct {
 //
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.14.1/pkg/reconcile
-func (r *HorizonTestReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+func (r *HorizonTestReconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ctrl.Result, _err error) {
```

Comment:
Was this change needed because of `golangci|operator-lint`? I'm just curious:). 

This could have been a separate commit together with the renaming of the variable `result` -> `yamlResult`. It's ok to keep it as is right now. Just something to keep in mind.

Reply:
To keep code across all operators consistent I added the _err error, because it was needed in the PatchInstance function. And then I needed to add result as well, because named and unnamed type can't be combined. And that resulted in changing result to yamlResult :sweat_smile: But you are right, it would have been better as it's own commit. I can try to make it a different commit by playing with git.